infrastructure/containers/postgres/automatic-backup/backup_or_restore.py

```diff
@@ -85,32 +130,43 @@ def backup_database_on_interval(seconds):
     )
     while True:
         time.sleep(seconds)
-        backup_database(make_backup_fpath())
+        backup_database(make_object_name())
```

Comment:
```suggestion
        s3_object_name: str = make_object_name()
        backup_database(s3_object_name)
```

Reply:
Added this change and changed the name of make_object_name to be less vague.